### PR TITLE
Feature/without periods

### PR DESCRIPTION
**Pull request by scientistnik**

### CODE REVIEW

File: contracts/Association.sol [hunk 1]
```diff
@@ -33,21 +33,20 @@ contract Association is Ownable {
...
         ATTACH_TOKEN,
         SET_BANK_ADDRESS,
-        RELEVANCE_PERIOD,
-        QUEUE_PERIOD,
         SET_FEES,
         ADD_ORACLE,
         DISABLE_ORACLE,
         ENABLE_ORACLE,
         DELETE_ORACLE,
         SET_SCHEDULER,
```
Comment (decodedbrain) [inline contracts/Association.sol]: Delete.

Comment (decodedbrain) [reply contracts/Association.sol]: Also delete var and method

File: contracts/Association.sol [hunk 2]
```diff
@@ -7,7 +7,7 @@ import "./token/LibreCash.sol";
 import "./ComplexBank.sol";
 
 /**
- * The shareholder association contract itself
+ * The shareholder association contract itsel
```
Comment (decodedbrain) [inline contracts/Association.sol]: Replace back

File: contracts/Association.sol [hunk 3]
```diff
@@ -298,22 +294,6 @@ contract Association is Ownable {
...
-        return bank.getBuyOrder(_orderID);
-    }
-
-    function getSellOrder(uint256 _orderID) public onlyShareholders view 
-        returns (address, address, uint256, uint256, uint256) 
-    {
-        return bank.getSellOrder(_orderID);
-    }
-
     function readyOracles() public onlyShareholders view returns (uint256) {
```
Comment (decodedbrain) [inline contracts/Association.sol]: Delete it and make public in bank

File: contracts/ComplexBank.sol [hunk 4]
```diff
@@ -560,7 +231,7 @@ contract ComplexBank is Pausable, BankI {
         for (address current = firstOracle; current != 0x0; current = oracles[current].next) {
             if (!oracles[current].enabled) 
                 continue;
-            if (OracleI(current).waitQuery() && (now - timeUpdateRequest) < oracleTimeout) {
+            if (OracleI(current).waitQuery() && (now - requestTime) < oracleTimeout) {
```
Comment (decodedbrain) [inline contracts/ComplexBank.sol]: Or use request time from oracle?

File: contracts/ComplexBank.sol [hunk 5]
```diff
@@ -569,30 +240,36 @@ contract ComplexBank is Pausable, BankI {
...
 
     /**
-     * @dev Lets owner to set queue period.
-     * @param _period Period from MIN_QUEUE_PERIOD.
+     * @dev Lets owner to set  Oracle actual period.
+     * @param _period Oracle data actual timeout.
      */
-    function setQueuePeriod(uint256 _period) public onlyOwner {
-        queuePeriod = _period;
+    function setOracleActual(uint256 _period) public onlyOwner {
```
Comment (decodedbrain) [inline contracts/ComplexBank.sol]: Maybe add some require() validations

File: contracts/ComplexBank.sol [hunk 6]
```diff
@@ -740,16 +417,9 @@ contract ComplexBank is Pausable, BankI {
...
-                if (buyOrders[i].recipient != 0x0) {
-                    reserveBalance = reserveBalance.sub(buyOrders[i].amount);
-                }
-            }
-            reserveBalance = reserveBalance.sub(overallRefundValue);
-            uint256 canGetCryptoBySellingTokens = (token.totalSupply() * RATE_MULTIPLIER) / cryptoFiatRateSell;
-            reserve = (reserveBalance * REVERSE_PERCENT * 100) / canGetCryptoBySellingTokens;
+        if ((curBalance != 0) && (sellRate != 0)) {
+            uint256 canGetCryptoBySellingTokens = (reserveTokens * RATE_MULTIPLIER) / sellRate;
+            reserve = (curBalance * FEE_MULTIPLIER * 100) / canGetCryptoBySellingTokens;
```
Comment (decodedbrain) [inline contracts/ComplexBank.sol]: Mayber rename `canGetCryptoBySellingTokens` to shorter verion.

File: contracts/Association.sol (hunk 3 above)
Comment (scientistnik) [reply contracts/Association.sol]: how did this get in here?!

**Pull request merged**